src/main/webapp/dev/js/common/visibilityOptions.es6

```diff
@@ -373,6 +373,9 @@ function updateVisibilityMessageDiv($containingForm) {
         type: 'POST',
         url,
         data: formData,
+        beforeSend() {
+            $visibilityMessageDiv.html("<img height='25' width='25' src='/images/ajax-loader.gif'/>");
```

Comment:
The dimensions are not 25x25 though (I think it's okay to completely omit the width and height).